TravelFunds/2019.md

```diff
@@ -3,7 +3,7 @@
 First Name | Last Name | Event | Reason | Trip Report | Location | Travel Dates | Amount Requested: | Pull Request date | Pull Request link | Date Expense report sent | Amount of Expense Report | Date Sent to Finance | Date approved through Bill.com | Bill.com Amount approved for reimbursement
 -|-|-|-|-|-|-|-|-|-|-|-|-|-|-
 Anna | Henningsen | Diagnostics WG Summit | Attendance | N/A | Munich |6 Mar – 9 Mar 2019 | 252.68 € | 21 Jan 2019 | https://github.com/nodejs/admin/pull/295 ||$288.56|22 Mar 2019||$288.56
-Rich | Trott | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | US $1404.12 | 8 Mar 2019 | https://github.com/nodejs/admin/pull/309 |||||
+Rich | Trott | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | US $1404.12 | 8 Mar 2019 | https://github.com/nodejs/admin/pull/309 ||$1,404.12|2 Jul 2019||
```

Comment:
I believe this change is unrelated to the other changes in this branch, but it LGTM anyway.

Reply:
Yes, sorry, user error - I pushed to the wrong branch without thinking...

Reply:
I reverted this commit last night, and will push it in a separate branch.